pillar/vector/mitx.sls

```diff
@@ -0,0 +1,371 @@
+vector:
+  configuration:
+
+    api:
+      enabled: true
+
+    log_schema:
+      timestamp_key: "@timestamp"
+
+    sources:
+
+      {% if 'edx' in salt.grains.get('roles') %}
+
+      nginx_access_log:
+        type: file
+        include:
+          - /edx/var/log/nginx/access.log
+
+      nginx_error_log:
+        type: file
+        include:
+          - /edx/var/log/nginx/error.log
+
+      # This is gone. Changed in Koa?
+      # cms_log:
+      #   type: file
+      #   include:
+      #     - /edx/var/log/cms/edx.log
+
+      cms_stderr_log:
+        type: file
+        include:
+          - /edx/var/log/supervisor/cms-stderr.log
+
+      # This is gone. Changed in Koa?
+      # lms_log:
+      #   type: file
+      #   include:
+      #     - /edx/var/log/lms/edx.log
+
+      lms_stderr_log:
+        type: file
+        include:
+          - /edx/var/log/supervisor/lms-stderr.log
+
+      {% endif %}
+
+      {% if 'edx-worker' in salt.grains.get('roles') %}
+
+      worker_cms_stderr_log:
+        type: file
+        include:
+          - /edx/var/log/supervisor/cms_*stderr.log
+
+      worker_lms_stderr_log:
+        type: file
+        include:
+          - /edx/var/log/supervisor/lms_*stderr.log
+
+      {% endif %}
+
+      tracking_log:
```

Comment:
I think this should go under the `edx` role

Reply:
I had applied this to both the `edx` and `edx-worker` roles (i.e. outside of the conditionals) because a `find` command reveals that they exist on both. `find /edx/var/log/tracking -name "*.log" -mtime -1 -ls` shows this log on one of the production `edx-worker-mitx-production-residential-live` minions and on one of the `draft` instances. Do you think I should ignore them?